src/containers/auth/register/Register.test.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable testing-library/no-unnecessary-act */
+/* eslint-disable testing-library/prefer-screen-queries */
+/* eslint-disable testing-library/render-result-naming-convention */
+
+import {
+  render,
+  fireEvent,
+  screen,
+  act,
+  waitFor,
+} from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+
+import Register from "./Register";
+import { AuthContext } from "../AuthProvider";
+const defaultProps = {};
+
+const mockRegisterUserFlow = jest.fn();
+
+const defaultContextValues = {
+  isProcessing: false,
+  hasPermission: jest.fn(() => true),
+  setAuth: jest.fn(),
+  setIsBackendAuthorized: jest.fn(),
+  registerUserFlow: mockRegisterUserFlow,
+};
+
+const setup = (props = {}, contextValues = {}) => {
+  const setupProps = { ...defaultProps, ...props };
+  const setupContextValues = { ...defaultContextValues, ...contextValues };
+  return render(
+    <BrowserRouter>
+      <AuthContext.Provider value={setupContextValues}>
+        <Register {...setupProps} />
+      </AuthContext.Provider>
+    </BrowserRouter>
+  );
+};
+
+describe("Register component tests", () => {
+  test("Renders the Login component", () => {
+    setup();
+    const signUpHeader = screen.getByTestId("signup-button");
```

Comment:
you are not receiving the the correct header here.

Reply:
Fixed